release/rancher/rancher.go

```diff
@@ -149,6 +150,47 @@ func GeneratePrimeArtifactsIndex(path string, ignoreVersions []string) error {
 	return os.WriteFile(filepath.Join(path, "index-prerelease.html"), preReleaseIndex, 0644)
 }
 
+func UpdateDashboardReferences(ctx context.Context, ghClient *github.Client, r *config.DashboardRelease, u *config.User) error {
+	if err := updateDashboardReferencesAndPush(r, u); err != nil {
+		return err
+	}
+
+	if r.DryRun {
```

Comment:
The dry run functionality is meant for the CLI only code and shouldn't leak into the package implementation code.

Reply:
Removed it.